src/app/lib/config/toggles/liveConfig.js

```diff
@@ -0,0 +1,58 @@
+export default {
+  _environment: {
```

Comment:
whats the requirement behind this key?

Reply:
Purely for testing purposes - https://github.com/bbc/simorgh/blob/809b25c2ab143fb50a8586c7e9c876de869be934/src/app/lib/config/toggles/index.test.js#L10 - I had added a comment in a previous revision but removed it.